src/components/AnonymousChat/child/ChatItemText.style.ts

```diff
@@ -142,6 +142,60 @@ export const styles = StyleSheet.create({
   },
   radius8: {
     borderRadius: 8
+  },
+  attachmentContainer: {
+    width: '100%',
+    height: 268,
+    borderRadius: 8,
+    overflow: 'hidden',
+    flexDirection: 'row',
+    flexWrap: 'wrap'
+  },
+  image: {
+    width: '100%',
+    height: '100%',
+    resizeMode: 'cover'
+  },
+  moreOverlay: {
+    position: 'absolute',
+    backgroundColor: 'rgba(0,0,0,0.5)',
+    width: '100%',
+    height: '100%',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  moreText: {
+    fontSize: dimen.normalizeDimen(16),
+    fontFamily: fonts.inter[400],
+    color: colors.white
+  },
+  attachmentFileContainer: {
+    backgroundColor: colors.alto,
+    justifyContent: 'center',
+    minHeight: dimen.normalizeDimen(64),
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  attachmentFileContent: {
+    flex: 1,
+    padding: dimen.normalizeDimen(6)
+  },
+  attachmentFileName: {
+    fontSize: dimen.normalizeDimen(14),
+    fontFamily: fonts.inter[600],
+    color: colors.black
+  },
+  attachmentFileInfo: {
+    fontSize: dimen.normalizeDimen(12),
+    fontFamily: fonts.inter[400],
+    color: colors.gray
+  },
+  attachmentFileIcon: {
+    backgroundColor: colors.light_silver,
+    width: dimen.normalizeDimen(64),
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new styles for handling attachments in the chat look good. Just ensure that these dimensions work well across different screen sizes and resolutions.


<!-- This is an auto-generated comment by OSS CodeRabbit -->